drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -555,6 +555,18 @@ config NET_TC_TX_COUNT
 
 endif # NETWORKING
 
+choice NET_PKT_DATA_ALLOC_TYPE
+	default NET_BUF_VARIABLE_DATA_SIZE
+endchoice
+
+if NET_BUF_VARIABLE_DATA_SIZE
+config NET_PKT_BUF_RX_DATA_POOL_SIZE
+	default 32000
+
+config NET_PKT_BUF_TX_DATA_POOL_SIZE
+	default 55000
+endif
```

Comment:
Please add some comments as to how we have arrived at these numbers and if there is specific config or results (throughput).